include/CalledMethods.h

```diff
@@ -6,12 +6,13 @@
 class CalledMethods : public DataflowPass {
   protected:
     void leastUpperBound(PVAliasSet &preSet, MethodsSet &curMethodsSet);
-    void onAllocationFunctionCall(PVAliasSet *input, std::string &fnName);
-    void onDeallocationFunctionCall(PVAliasSet *input, std::string &fnName);
-    void onUnknownFunctionCall(PVAliasSet *input);
-    void onReallocFunctionCall(PVAliasSet *input, std::string &fnName);
-    void onSafeFunctionCall(PVAliasSet *input, std::string &fnName);
-    void onAnnotation(PVAliasSet *input, std::string &fnName,
+    void onAllocationFunctionCall(PVAliasSet* input, std::string &fnName);
+    void onDeallocationFunctionCall(PVAliasSet* input, std::string &fnName);
+    void onUnknownFunctionCall(PVAliasSet* input);
+    void onReallocFunctionCall(PVAliasSet* input, std::string &fnName);
+    void onSafeFunctionCall(PVAliasSet* input, std::string &fnName);
+    void onAnnotation(PVAliasSet* input, std::string &fnName, AnnotationType annotationType);
+    void onAnnotation(PVAliasSet* input, std::string &annoFnName, std::string& invokerFnName,
```

Comment:
When there are two methods with the same name but different parameter lists in a header file, it's a good idea to include a comment indicating the difference between the two. Can you do that here? The goal is that someone reading this file should be able to figure out which of the two is the right one to use in a given situation without needing to read the .cpp file.